### DISCUSSION TITLE
Code Builder Should also help creating the Entity Relationship Diagram (ERD)

### DISCUSSION THREAD

**Discussion by matteysumit** [forcedotcom/salesforcedx-vscode#5478] [Ideas]
**Is your feature request related to a problem? Please describe.**
The CODE Browser should have the capability of designing the Entity Relationship Diagram like the way we do in LUCID Charts. Entity relationship diagrams provide a visual starting point for database design that can also be used to help determine information system requirements throughout an org.

**Describe what you are ultimately trying to do**
I need to create the ERD Diagram of a complex org but the CODE BUilder doesn't have this feature. I need to use MS Visio or LUCID Chart.

**Describe how you think Code Builder can help solve your problem or improve your work**
If the ERD Diagram feature is available in Code Builder then no developer or architect needs to step out of the application.

**Comment by smaddox-sf** [comment]
Hi @matteysumit - Thanks for trying out Code Builder.  As a bit of internal housekeeping, I am transferring this idea to our IDE Experience repo.

**Comment by AnanyaJha** [comment]
Hi @matteysumit, thank you for sharing this feedback! We'll keep this in mind for our future roadmap
